include/sealib/trailstructure.h

```diff
@@ -5,7 +5,7 @@
 #include <sealib/rankselect.h>
 #include <sealib/simplerankselect.h>
 #include <sealib/dyckmatchingstructure.h>
-#include <sealib/largedoublelinkedlist.h>
+#include <src/largedoublelinkedlist.h>
```

Comment:
You can't reference this private header file from a public header file. All the end user has access to is the `include` directory and and the library file, not the `src` directory.